lib/pure/collections/critbits.nim

```diff
@@ -342,10 +333,11 @@ iterator leaves[T](n: Node[T]): Node[T] =
 iterator keys*[T](c: CritBitTree[T]): string =
   ## Yields all keys in lexicographical order.
   runnableExamples:
-    let c = {"key1": 1, "key2": 2}.toCritBitTree
-    var keys: seq[string]
-    for key in c.keys:
-      keys.add(key)
+    from sequtils import toSeq
+
+    let
+      c = {"key1": 1, "key2": 2}.toCritBitTree
+      keys = toSeq(c.keys)
     doAssert keys == @["key1", "key2"]
```

Comment:
avoid variables used just once, prefer concise runnableExamples, eg:
```nim
from sequtils import toSeq
let c = {"key1": 1, "key2": 2}.toCritBitTree
doAssert toSeq(c.keys) == @["key1", "key2"]
```

(main difference is avoiding intermediate `keys` variable)

ditto elsewhere